docs/root/faq/edge.rst

```diff
@@ -0,0 +1,85 @@
+.. _faq_edge:
+
+Can I use Envoy as an edge proxy?
+=================================
+
+Envoy is a production-ready edge proxy, and performs well in that role, however,
+the default settings are tailored for the service mesh use case, and some values
+need to be adjusted when using Envoy as an edge proxy.
+
+TCP proxies should configure: :ref:`overload_manager <config_overload_manager>`,
+:ref:`listener buffer limits <envoy_api_field_Listener.per_connection_buffer_limit_bytes>` to 32,768 bytes
+and :ref:`cluster buffer limits <envoy_api_field_Cluster.per_connection_buffer_limit_bytes>` to 32,768 bytes.
+
+HTTP proxies should additionally configure:
+:ref:`use_remote_address <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.use_remote_address>` to true,
+and HTTP/2 settings (e.g.
+:ref:`HTTP/2 maximum concurrent streams limit <envoy_api_field_core.Http2ProtocolOptions.max_concurrent_streams>` to 100,
+:ref:`HTTP/2 initial stream window size limit <envoy_api_field_core.Http2ProtocolOptions.initial_stream_window_size>` to 65,535 bytes,
+and :ref:`HTTP/2 initial connection window size limit <envoy_api_field_core.Http2ProtocolOptions.initial_connection_window_size>` to 1,048,576 bytes).
+
+The following is a YAML example of the above recommendation.
+
+.. code-block:: yaml
+
+  overload_manager:
+    refresh_interval:
```

Comment:
I think you can just write things like `refresh_interval: 0.25s` in YAML, there is special WKT treatment of durations in the JSON -> protobuf ingestion.

Reply:
Done.